permissions/api_permissions.py

```diff
@@ -170,7 +170,9 @@ def has_permission(self, request, view):
         if request.method in permissions.SAFE_METHODS:
             return True
 
-        service_sector_id = request.data.get("service_sector_id")
+        service_sector_id = request.data.get("service_sector_id", None)
+        if not service_sector_id and view.detail:
```

Comment:
Hmm, what's the case where we need this special case?

Reply:
When updating only single field (round's status), even though its detail it will bump first here and then go to the object permission check. TBH, I'm no sure for the whole existing service_sector check. But perhaps that's something to consider in the permission refactor.